lib/email-validation.rb

```diff
@@ -3,8 +3,12 @@
 require 'email-validation/config'
 
 module EmailValidation
-  def self.incorrect_email_message(email)
-    "The email address you have entered (#{email}) is incorrect."
+  def self.invalid_email_message
```

Comment:
what you wrote is ok but I added small helper method that returns contact link html, and then if you add method call to the constant string, it doesn't sound right for me.